lib/toolResponse.types.ts

```diff
@@ -3,4 +3,7 @@ export enum CreateTokenResponse {
   MISSING_TITLE = "MISSING_TITLE",
   MISSING_COLLECTION = "MISSING_COLLECTION",
   SIGN_TRANSACTION = "SIGN_TRANSACTION",
+  MISSING_AUTH = "MISSING_AUTH",
+  MISSING_WALLET = "MISSING_WALLET",
```

Comment:
What's the difference between `MISSING_AUTH` & `MISSING_WALLET`?
- When will they be different?